packages/components/ovh-at-internet/src/utils.ts

```diff
@@ -1,6 +1,44 @@
 export function debug(...args: any[]) {
-  if (!window.test && //temporarily  fix jest bug
-    window.localStorage?.getItem('MANAGER_TRACKING_DEBUG')) {
+  if (
+    !window.test && // temporarily  fix jest bug
+    window.localStorage?.getItem('MANAGER_TRACKING_DEBUG')
+  ) {
     console.debug(...args);
   }
 }
+
+export const isEmpty = (obj: any) => {
+  if (obj?.length || obj?.size) return false;
+  if (typeof obj !== 'object') return true;
+  return !Object.values(obj).length;
+};
```

Comment:
Can you please add comments regarding the goal of this function ?

Reply:
I commented the functions that were added